docs/install/install-aur.md

```diff
@@ -6,6 +6,37 @@ sidebar_position: 4
 
 # Installing Vignette using AUR
 
-something
+If you wish to install Vignette on your Arch Linux computer, you may use any AUR helper of you choice. The package is [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/). In the example below, I will be using yay.
+
+
+
+## Installation process
+
+In order to install Vignette using yay, please run the following command: 
+
+`yay vignette-git`
+
+
+
+If more than one package appears you should find the right package, which should be listed as `aur/vignette-git`. In this example, that package is number `1`.
```

Comment:
hmm like this?

```suggestion
If more than one package appears you should find the right package, which should be listed as `aur/vignette-git`. Here, that package is number `1`.
```

Reply:
"in this example" phrasing is consistent with the rest of the page